app/src/main/java/com/android/feedme/model/viewmodel/CameraViewModel.kt

```diff
@@ -324,4 +315,18 @@ class CameraViewModel : ViewModel() {
       _listOfIngredientToInput.value += ing
     }
   }
+
+  /**
+   * This function is used to empty the error and information to display after it has been
+   * processed.
+   */
+  fun empty() {
+    _errorToDisplay.value = null
+    _informationToDisplay.value = null
+  }
+
+  /** This function is used to empty the list of ingredients after it has been processed. */
+  fun emptyIngredients() {
+    _listOfIngredientToInput.value = emptyList()
+  }
```

Comment:
Nice idea to add inner boolean values, to be sure nothing is executed before the screen is displayed.